lib/rubocop/cop/style/character_literal.rb

```diff
@@ -32,13 +32,11 @@ def autocorrect(node)
 
         # Dummy implementation of method in ConfigurableEnforcedStyle that is
         # called from StringHelp.
-        def opposite_style_detected
-        end
+        def opposite_style_detected; end
```

Comment:
By the way `;` is not really needed here. Maybe we should have two styles for enforcing its presence or absence.